src/Statistics.jl

```diff
@@ -958,6 +958,8 @@ function quantile!(v::AbstractVector, p::Union{AbstractArray, Tuple{Vararg{Real}
     end
     return map(x->_quantile(v, x, alpha=alpha, beta=beta), p)
 end
+quantile!(arr::AbstractArray, p::Union{AbstractArray,Tuple{Vararg{Real}}}; sorted::Bool=false, alpha::Real=1.0, beta::Real=alpha) =
+    quantile!(arr[:], p, sorted=sorted, alpha=alpha, beta=alpha)
```

Comment:
Shouldn't make a copy since this function is supposed to be in-place:
```suggestion
quantile!(arr::AbstractArray, p::Union{AbstractArray,Tuple{Vararg{Real}}};
          sorted::Bool=false, alpha::Real=1.0, beta::Real=alpha) =
    quantile!(vec(arr), p, sorted=sorted, alpha=alpha, beta=alpha)
```